### PR TITLE
Fixes #2093: Add check of the number of connections in couchbase test

### LINKED ISSUE #2093: Add check of the number of connections in couchbase test
Add check of the number of connections in `CouchbaseIT.java` test, similarly to `MongoDbTest.java`.

### PR DESCRIPTION
**Pull request by vga91**
Fixes #2093

### CODE REVIEW

File: full/src/test/java/apoc/couchbase/CouchbaseIT.java [hunk 1]
```diff
@@ -49,6 +53,18 @@ public static void tearDown() {
...
+    @Before
+    public void before() {
+        numberConnections = getNumConnections();
+    }
+
+    @After
+    public void after() {
+        // the connections active before must be equal to the connections active after
+        long numConnectionsAfter = getNumConnections();
+        assertEquals(numberConnections, numConnectionsAfter);
```
Comment (jexp) [inline full/src/test/java/apoc/couchbase/CouchbaseIT.java:66]: what happens if not and what would be the reason for not?

**Review by jexp — APPROVED** [review]
Looks good but I don't fully understand the why

File: full/src/test/java/apoc/couchbase/CouchbaseIT.java (hunk 1 above)
Comment (vga91) [reply full/src/test/java/apoc/couchbase/CouchbaseIT.java:66]: For example if I don't wrap correctly the procedure in a `try-res`, and so I don't call the `apoc.java.CouchbaseConnection.close()` method to disconnect the `Cluster` and shutdown the `ClusterEnvironment`, 
I receive a `numberConnections` equals to `numConnectionsAfter + 1`.
So, this assert is useful to verify that you have put the `try-res` wherever it's needed

**Comment by conker84** [comment]
As we discussed in the last meeting, is for checking that we close any connection to couchbase properly as we did for the mongo procedures

**Pull request merged**